packages/block-library/src/site-title/edit/index.js

```diff
@@ -19,7 +19,7 @@ import { createBlock, getDefaultBlockName } from '@wordpress/blocks';
 /**
  * Internal dependencies
  */
-import LevelToolbar from './level-toolbar';
+import LevelControl from './level-toolbar';
```

Comment:
should we rename the file :). This PR looks good otherwise.

Reply:
No need, because I'm planning on creating a PR today for extracting `HeadingLevelDropdown` dropdown and use it in this block too.